renew-cert.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# fail on error
+set -x
+
+docker exec acme-sh --renew -d monitoring-test.localzero.net -d monitoring.localzero.net \
```

Comment:
Good point. Acme.sh has a lot of options and pre-automated features, and it took me a while to find a good way to make it work with our situation and without mounting the docker socket into the acme.sh container since that's a huge security risk.
I knew they had their own cronjob somehow, but wasn't sure it would work with our files. The logs look like it works though; so I will remove our own cronjob again.

Reply:
Good point. Acme.sh has a lot of options and pre-automated features, and it took me a while to find a good way to make it work with our situation and without mounting the docker socket into the acme.sh container (so it can tell nginx to reload its config) since that's a huge security risk.
I knew they had their own cronjob somehow, but wasn't sure it would work with our files. The logs look like renewal itself works, but the problem is that nginx needs to reload its config afterwards.

I see three options:
1. Change our own cronjob so it just reloads the nginx config twice a day and leave renewal to acme.sh. Since acme.sh renews every 60 days but the cert is valid 90 days, there is no risk of the cert expiring before the next nginx reload.
2. Keep everything as it is. Disadvantage: two things handle renewal, difficult to understand.
3. Run acme.sh not as a daemon but only as a cronjob. Disadvantage: container run options have to be defined in two places, cronjob and compose file.

Option 1 seems preferable to me. All options should also work after we add the top level nginx instance.

Reply:
I'll test next weekend if acme.sh's own cron job does all we need, and if so, will implement option 1 unless you disagree.